<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormUsuarios</class>
 <widget class="QMainWindow" name="FormUsuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Usuários</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../img/cliente.png</normaloff>../../img/cliente.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Usuário:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPesquisaUsuario">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableCliente">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>321</width>
      <height>301</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <column>
     <property name="text">
      <string>Cód. Usuário</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Usuário</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>15</y>
      <width>166</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnPesquisarUsuario">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pesquisar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../img/lupa.png</normaloff>../../img/lupa.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLimpar">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Limpar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../img/borracha.png</normaloff>../../img/borracha.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>415</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnExcluirUsuario">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Excluir Usuário</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../img/excluircliente.png</normaloff>../../img/excluircliente.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnAlterarUsuario">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Alterar Usuário</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../img/editar usuario.png</normaloff>../../img/editar usuario.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnCadastrarUsuario">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cadastrar Usuário</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../img/cadastrar usuario.png</normaloff>../../img/cadastrar usuario.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>editPesquisaUsuario</tabstop>
  <tabstop>btnPesquisarUsuario</tabstop>
  <tabstop>btnLimpar</tabstop>
  <tabstop>tableCliente</tabstop>
  <tabstop>btnAlterarUsuario</tabstop>
  <tabstop>btnCadastrarUsuario</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnLimpar</sender>
   <signal>clicked()</signal>
   <receiver>editPesquisaUsuario</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
